README.md

```diff
@@ -17,7 +17,7 @@ The default admin username and password can be found in vars.tf under `admin_nam
 <i>Note: <b>EasyKey</b> from the output will contain configuration that can be applied to Spoke VPN devices for ease of configuration. </i>
```

Comment:
```suggestion
> **Note:** For ease of configuration, search for **EasyKey** in the output. It will contain configuration that can be applied to Spoke VPN devices.
```